packages/pagination/src/Pagination.story.tsx

```diff
@@ -20,6 +20,8 @@ export default {
     darkMode: storybookArgTypes.darkMode,
     onBackArrowClick: { control: false },
     onForwardArrowClick: { control: false },
+    onCurrentPageOptionChange: { control: false },
+    onItemsPerPageOptionChange: { control: false },
```

Comment:
Since we have so many excluded props, it's probably more readable to use `parameters.controle.exclude` syntax here